map_mutation_class.py

```diff
@@ -0,0 +1,51 @@
+"""
+Gregory Way 2017
+PanCancer Classifier
+map_mutation_class.py
+
```

Comment:
At the beginning of each script, it would be helpful to describe what the script does in general.